pkgs/by-name/hi/himalaya/package.nix

```diff
@@ -3,65 +3,71 @@
 , fetchFromGitHub
 , stdenv
 , pkg-config
-, darwin
+, apple-sdk
 , installShellFiles
 , installShellCompletions ? stdenv.buildPlatform.canExecute stdenv.hostPlatform
 , installManPages ? stdenv.buildPlatform.canExecute stdenv.hostPlatform
 , notmuch
 , gpgme
-, buildNoDefaultFeatures ? false
-, buildFeatures ? []
+, withDefaultFeatures ? false
+, withFeatures ? []
 }:
 
-rustPlatform.buildRustPackage rec {
-  # Learn more about available cargo features at:
-  #  - <https://pimalaya.org/himalaya/cli/latest/installation.html#cargo>
-  inherit buildNoDefaultFeatures buildFeatures;
+let
+  version = "1.0.0";
+  hash = "sha256-NpkjnU4WBHnXHBoFqkMmX1l2xb038dDYsvWjNnpzBnM=";
+  cargoHash = "sha256-pIrNSCqlRsx2QlknSVSGvkfoHenpzfTd1AAVywBzdpo=";
+in
+
+rustPlatform.buildRustPackage {
+  inherit version cargoHash;
 
   pname = "himalaya";
-  version = "1.0.0-beta.4";
 
   src = fetchFromGitHub {
-    owner = "soywod";
-    repo = pname;
+    inherit hash;
+    owner = "pimalaya";
+    repo = "himalaya";
     rev = "v${version}";
-    hash = "sha256-NrWBg0sjaz/uLsNs8/T4MkUgHOUvAWRix1O5usKsw6o=";
   };
 
-  cargoHash = "sha256-YS8IamapvmdrOPptQh2Ef9Yold0IK1XIeGs0kDIQ5b8=";
-
-  NIX_LDFLAGS = lib.optionals stdenv.hostPlatform.isDarwin [
-    "-F${darwin.apple_sdk.frameworks.AppKit}/Library/Frameworks"
-    "-framework"
-    "AppKit"
-  ];
+  buildNoDefaultFeatures = withDefaultFeatures;
+  buildFeatures = withFeatures;
 
   nativeBuildInputs = [ pkg-config ]
-    ++ lib.optional (builtins.elem "pgp-gpg" buildFeatures) pkg-config
```

Comment:
It'd be probably be better to remove `pkg-config` from the first list here, as it doesn't seem to be required unless `"pgp-gpg"` or `"notmuch"` are set as build features

Reply:
If I do not mistake, `pkg-config` is also needed by other subcrates (like `ring`), so the previous implementation was incomplete. Better to keep `pkg-config` as default.